pages/translations/en/LC_MESSAGES/messages.po

```diff
@@ -1147,7 +1150,1930 @@ msgstr ""
+msgid "Load pages in an instant"
+msgstr ""
+
+#: pages/content/amp-dev/about/how-amp-works.html:92
+msgid "The new <a href=\"http://www.w3.org/TR/resource-hints/#dfn-preconnect\">preconnect API</a> is used heavily to ensure HTTP requests are as fast as possible when they are made. With this, a page can be rendered before the user explicitly states they’d like to navigate to it; the page might already be available by the time the user actually selects it, leading to instant loading."
+msgstr ""
+
+#: pages/content/amp-dev/about/how-amp-works.html:93
+msgid "While prerendering can be applied to all web content, it can also use up a lot of bandwidth and CPU. AMP is optimized to reduce both of these factors. Prerendering only downloads resources above the fold and prerendering doesn’t render things that might be expensive in terms of CPU."
+msgstr ""
+
+#: pages/content/amp-dev/about/how-amp-works.html:95
+msgid "Learn more about <a href=\"https://medium.com/@cramforce/why-amp-html-does-not-take-full-advantage-of-the-preload-scanner-7e7f788aa94e\">why AMP HTML doesn’t take full advantage of the preload scanner</a>."
+msgstr ""
+
+#: pages/content/amp-dev/index.html:370
+msgid "Developer Benefits"
+msgstr ""
+
+#: pages/content/amp-dev/about/ads.html:250
+msgid "Regular Ad"
+msgstr ""
+
+msgid "JavaScript is powerful, it can modify just about every aspect of the page, but it can also block DOM construction and delay page rendering (see also <a href=\"https://developers.google.com/web/fundamentals/performance/critical-rendering-path/adding-interactivity-with-javascript\">Adding interactivity with JavaScript</a>). To keep JavaScript from delaying page rendering, AMP allows only asynchronous JavaScript."
+msgstr ""
+
+msgid "AMP uncouples document layout from resource layout. Only one HTTP request is needed to layout the entire doc (<a href=\"#font-triggering-must-be-efficient\">+fonts</a>). Since AMP is optimized to avoid expensive style recalculations and layouts in the browser, there won’t be any re-layout when resources load."
+msgstr ""
+
+msgid "Any page that uses a custom script must tell the AMP system that it will eventually have a custom tag. For example, the <a href=\"../documentation/components/reference/amp-iframe.md\"><code>amp-iframe</code></a> script tells the system that there will be an <code>amp-iframe</code> tag. AMP creates the iframe box before it even knows what it will include:"
+msgstr ""
+
+msgid "AMP controls all resource downloads: it prioritizes resource loading, loading only what’s needed, and prefetches lazy-loaded resources."
+msgstr ""
+
+msgid "When AMP documents get prerendered for instant loading, only resources above the fold are actually downloaded. When AMP documents get prerendered for instant loading, resources that might use a lot of CPU (like third-party iframes) do not get downloaded."
+msgstr ""
+
+msgid "Vision & Mission"
+msgstr ""
+
+msgid "Vision and Mission"
```

Comment:
s/and/& ?

Reply:
VIsion & Mission is used in a large hero background, Vision and Mission was used in section headings. In general, I didn't try to make judgement calls 

![image](https://user-images.githubusercontent.com/465414/74040659-f7c4e280-4978-11ea-80a5-a52768e15330.png)

![image](https://user-images.githubusercontent.com/465414/74040684-04493b00-4979-11ea-83a2-aaba2ce033fb.png)

more than happy to make that change, just want to confirm the effect beforehand

Reply:
Let's make this consistent. The page itself also uses ampersand.